2020-05-25 20:30:15,010 Exception on /psiturk [GET]
Traceback (most recent call last):
  File "/home/adat/.local/lib/python2.7/site-packages/flask/app.py", line 2317, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/adat/.local/lib/python2.7/site-packages/flask/app.py", line 1840, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/adat/.local/lib/python2.7/site-packages/flask/app.py", line 1743, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/adat/.local/lib/python2.7/site-packages/flask/app.py", line 1838, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/adat/.local/lib/python2.7/site-packages/flask/app.py", line 1824, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/adat/.local/lib/python2.7/site-packages/psiturk/experiment.py", line 747, in regularpage
    return render_template(pagename)
  File "/home/adat/.local/lib/python2.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/adat/.local/lib/python2.7/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/adat/.local/lib/python2.7/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/adat/.local/lib/python2.7/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/adat/.local/lib/python2.7/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/adat/.local/lib/python2.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/home/adat/.local/lib/python2.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
TemplateNotFound: psiturk
2020-05-25 20:31:16,138 Exception on /psiturk [GET]
Traceback (most recent call last):
  File "/home/adat/.local/lib/python2.7/site-packages/flask/app.py", line 2317, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/adat/.local/lib/python2.7/site-packages/flask/app.py", line 1840, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/adat/.local/lib/python2.7/site-packages/flask/app.py", line 1743, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/adat/.local/lib/python2.7/site-packages/flask/app.py", line 1838, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/adat/.local/lib/python2.7/site-packages/flask/app.py", line 1824, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/adat/.local/lib/python2.7/site-packages/psiturk/experiment.py", line 747, in regularpage
    return render_template(pagename)
  File "/home/adat/.local/lib/python2.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/adat/.local/lib/python2.7/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/adat/.local/lib/python2.7/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/adat/.local/lib/python2.7/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/adat/.local/lib/python2.7/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/adat/.local/lib/python2.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/home/adat/.local/lib/python2.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
TemplateNotFound: psiturk
2020-05-25 20:38:43,697 hit_assign_worker_id_not_set_in_mturk (1001) {}
2020-07-03 11:56:33,198 Exception on /screen.jpg [GET]
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/flask/app.py", line 2317, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1840, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1743, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1838, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/flask/app.py", line 1824, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/psiturk/experiment.py", line 747, in regularpage
    return render_template(pagename)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: screen.jpg
[2020-07-03 12:07:06 -0400] [9675] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-03 12:09:01 -0400] [9753] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-03 13:02:05 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:9906)
[2020-07-03 13:02:05 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:9913)
[2020-07-03 13:37:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:9907)
[2020-07-03 13:37:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:9908)
[2020-07-03 13:37:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:9909)
[2020-07-03 13:37:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:9910)
[2020-07-03 13:37:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:9911)
[2020-07-03 13:37:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:9912)
[2020-07-03 13:37:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10145)
[2020-07-03 13:37:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10146)
[2020-07-03 14:09:52 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:9914)
[2020-07-03 14:09:52 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10236)
[2020-07-03 14:09:52 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10237)
[2020-07-03 14:09:52 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10238)
[2020-07-03 14:09:52 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10240)
[2020-07-03 14:09:52 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10241)
[2020-07-03 14:09:52 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10242)
[2020-07-03 14:09:52 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10244)
[2020-07-03 14:56:30 -0400] [10270] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-03 16:40:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10243)
[2020-07-03 16:40:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10265)
[2020-07-03 16:40:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10268)
[2020-07-03 16:40:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10269)
[2020-07-03 16:40:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10271)
[2020-07-03 16:40:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10272)
[2020-07-03 16:40:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10273)
[2020-07-03 16:58:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10267)
[2020-07-03 16:58:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10270)
[2020-07-03 16:58:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10712)
[2020-07-03 16:58:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10713)
[2020-07-03 16:58:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10714)
[2020-07-03 16:58:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10715)
[2020-07-03 16:58:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10716)
[2020-07-03 16:58:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10717)
[2020-07-03 16:58:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10718)
[2020-07-03 18:14:10 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10728)
[2020-07-03 18:14:10 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10730)
[2020-07-03 18:14:10 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10731)
[2020-07-03 18:14:10 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10732)
[2020-07-03 18:14:10 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10735)
[2020-07-03 18:14:10 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10736)
[2020-07-03 18:48:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10729)
[2020-07-03 18:48:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10733)
[2020-07-03 18:48:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:10734)
[2020-07-03 18:48:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11084)
[2020-07-03 18:48:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11087)
[2020-07-03 18:48:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11090)
[2020-07-03 20:49:45 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11083)
[2020-07-03 20:49:45 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11085)
[2020-07-03 20:49:45 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11089)
[2020-07-03 20:49:45 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11202)
[2020-07-03 20:49:45 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11203)
[2020-07-03 20:49:45 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11205)
[2020-07-03 20:49:45 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11206)
[2020-07-03 20:49:45 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11207)
[2020-07-03 20:49:45 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11208)
[2020-07-03 21:49:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11227)
[2020-07-03 21:49:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11229)
[2020-07-03 21:49:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11231)
[2020-07-03 21:49:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11232)
[2020-07-03 21:49:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11233)
[2020-07-03 21:49:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11234)
[2020-07-03 21:49:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11235)
[2020-07-03 21:49:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11236)
[2020-07-03 21:49:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11237)
[2020-07-03 21:49:57 -0400] [11245] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-03 21:49:57 -0400] [11245] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-03 22:15:27 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11246)
[2020-07-03 22:24:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11244)
[2020-07-03 22:24:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11245)
[2020-07-03 22:24:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11247)
[2020-07-03 22:24:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11248)
[2020-07-03 22:24:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11249)
[2020-07-03 22:24:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11251)
[2020-07-03 22:24:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11252)
[2020-07-03 22:50:14 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11267)
[2020-07-03 22:50:14 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11304)
[2020-07-03 22:50:14 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11309)
[2020-07-03 22:50:17 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11308)
[2020-07-03 22:50:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11305)
[2020-07-03 22:50:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11306)
[2020-07-03 23:26:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11250)
[2020-07-03 23:26:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11303)
[2020-07-03 23:26:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11307)
[2020-07-03 23:26:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11543)
[2020-07-03 23:26:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11544)
[2020-07-03 23:26:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11546)
[2020-07-03 23:26:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11549)
[2020-07-03 23:26:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11550)
[2020-07-03 23:26:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11551)
[2020-07-03 23:26:31 -0400] [11561] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 00:16:51 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11556)
[2020-07-04 00:16:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11561)
[2020-07-04 00:16:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11564)
[2020-07-04 00:32:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11558)
[2020-07-04 00:32:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11560)
[2020-07-04 00:32:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11562)
[2020-07-04 00:32:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11563)
[2020-07-04 00:32:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11565)
[2020-07-04 00:32:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11566)
[2020-07-04 00:32:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11792)
[2020-07-04 00:32:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11796)
[2020-07-04 00:43:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11783)
[2020-07-04 00:43:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11840)
[2020-07-04 00:43:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11841)
[2020-07-04 00:43:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11843)
[2020-07-04 00:43:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11844)
[2020-07-04 00:43:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11845)
[2020-07-04 00:43:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11846)
[2020-07-04 00:43:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11847)
[2020-07-04 00:43:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11848)
[2020-07-04 00:51:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11864)
[2020-07-04 00:51:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11866)
[2020-07-04 00:51:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11867)
[2020-07-04 00:51:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11868)
[2020-07-04 00:51:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11869)
[2020-07-04 00:51:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11870)
[2020-07-04 00:51:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11871)
[2020-07-04 00:51:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11872)
[2020-07-04 01:00:32 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11865)
[2020-07-04 01:00:32 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11883)
[2020-07-04 01:00:32 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11884)
[2020-07-04 01:00:32 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11885)
[2020-07-04 01:00:32 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11886)
[2020-07-04 01:00:32 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11887)
[2020-07-04 01:00:32 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11888)
[2020-07-04 01:00:32 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11889)
[2020-07-04 01:00:32 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11890)
[2020-07-04 01:09:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11900)
[2020-07-04 01:09:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11904)
[2020-07-04 01:09:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11906)
[2020-07-04 01:09:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11907)
[2020-07-04 01:09:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11908)
[2020-07-04 01:09:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11909)
[2020-07-04 01:09:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11910)
[2020-07-04 01:09:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11911)
[2020-07-04 01:22:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11901)
[2020-07-04 01:22:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11938)
[2020-07-04 01:22:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11939)
[2020-07-04 01:22:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11940)
[2020-07-04 01:22:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11941)
[2020-07-04 01:22:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11942)
[2020-07-04 01:22:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11943)
[2020-07-04 01:22:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11944)
[2020-07-04 01:22:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11945)
[2020-07-04 01:22:10 -0400] [11956] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 01:22:10 -0400] [11956] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 01:31:02 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11954)
[2020-07-04 01:31:02 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11956)
[2020-07-04 01:31:02 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11958)
[2020-07-04 01:31:02 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11959)
[2020-07-04 01:31:02 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11960)
[2020-07-04 01:31:02 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11961)
[2020-07-04 01:31:02 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11962)
[2020-07-04 01:31:02 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11963)
[2020-07-04 01:31:02 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11964)
[2020-07-04 01:31:05 -0400] [11971] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 01:32:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11972)
[2020-07-04 01:32:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11977)
[2020-07-04 01:32:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11978)
[2020-07-04 01:32:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11979)
[2020-07-04 01:32:02 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11971)
[2020-07-04 01:32:02 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11976)
[2020-07-04 01:32:06 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11973)
[2020-07-04 01:34:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11974)
[2020-07-04 01:34:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11999)
[2020-07-04 01:34:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12000)
[2020-07-04 01:34:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12001)
[2020-07-04 01:34:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12002)
[2020-07-04 01:34:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12003)
[2020-07-04 01:34:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12004)
[2020-07-04 01:34:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12005)
[2020-07-04 01:37:40 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:11975)
[2020-07-04 01:37:40 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12033)
[2020-07-04 01:37:40 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12035)
[2020-07-04 01:37:40 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12036)
[2020-07-04 01:37:40 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12038)
[2020-07-04 01:37:40 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12040)
[2020-07-04 01:38:43 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12037)
[2020-07-04 01:38:43 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12057)
[2020-07-04 01:38:43 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12060)
[2020-07-04 01:46:09 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12034)
[2020-07-04 01:46:09 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12039)
[2020-07-04 01:46:09 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12054)
[2020-07-04 01:46:09 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12056)
[2020-07-04 01:46:09 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12058)
[2020-07-04 01:46:09 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12083)
[2020-07-04 01:46:09 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12084)
[2020-07-04 02:17:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12059)
[2020-07-04 02:17:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12085)
[2020-07-04 02:17:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12098)
[2020-07-04 02:17:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12100)
[2020-07-04 02:17:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12102)
[2020-07-04 02:17:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12103)
[2020-07-04 02:17:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12104)
[2020-07-04 02:17:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12105)
[2020-07-04 02:17:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12106)
[2020-07-04 02:48:10 -0400] [12137] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 04:09:21 -0400] [12137] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 06:11:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12128)
[2020-07-04 06:11:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12130)
[2020-07-04 06:11:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12131)
[2020-07-04 06:11:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12132)
[2020-07-04 06:11:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12133)
[2020-07-04 06:11:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12134)
[2020-07-04 06:11:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12135)
[2020-07-04 06:11:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12136)
[2020-07-04 06:18:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12137)
[2020-07-04 06:18:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12606)
[2020-07-04 06:18:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12607)
[2020-07-04 06:18:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12608)
[2020-07-04 06:18:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12609)
[2020-07-04 06:18:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12610)
[2020-07-04 06:18:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12611)
[2020-07-04 06:18:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12612)
[2020-07-04 06:18:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12613)
[2020-07-04 06:59:26 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12627)
[2020-07-04 06:59:26 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12628)
[2020-07-04 06:59:26 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12629)
[2020-07-04 06:59:26 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12631)
[2020-07-04 06:59:26 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12632)
[2020-07-04 06:59:26 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12633)
[2020-07-04 06:59:26 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12634)
[2020-07-04 06:59:26 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12635)
[2020-07-04 07:02:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12630)
[2020-07-04 07:02:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12653)
[2020-07-04 07:02:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12654)
[2020-07-04 07:02:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12655)
[2020-07-04 07:02:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12656)
[2020-07-04 07:02:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12657)
[2020-07-04 07:02:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12658)
[2020-07-04 07:02:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12659)
[2020-07-04 07:02:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12660)
[2020-07-04 09:04:22 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12689)
[2020-07-04 09:04:22 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12690)
[2020-07-04 09:04:22 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12691)
[2020-07-04 09:04:22 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12692)
[2020-07-04 09:04:22 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12693)
[2020-07-04 09:04:22 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12694)
[2020-07-04 09:04:22 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12695)
[2020-07-04 09:04:22 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12697)
[2020-07-04 09:04:22 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12698)
[2020-07-04 09:04:27 -0400] [12717] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 09:59:56 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12714)
[2020-07-04 09:59:56 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12715)
[2020-07-04 09:59:56 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12716)
[2020-07-04 09:59:56 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12717)
[2020-07-04 09:59:56 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12718)
[2020-07-04 09:59:56 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12721)
[2020-07-04 09:59:56 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12723)
[2020-07-04 11:00:00 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12719)
[2020-07-04 11:00:00 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12737)
[2020-07-04 11:00:00 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12738)
[2020-07-04 11:00:00 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12739)
[2020-07-04 11:00:00 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12740)
[2020-07-04 11:00:00 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12741)
[2020-07-04 11:00:00 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12742)
[2020-07-04 11:00:00 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12743)
[2020-07-04 11:38:15 -0400] [12769] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 11:39:54 -0400] [12768] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 12:47:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12722)
[2020-07-04 12:47:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12762)
[2020-07-04 12:47:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12765)
[2020-07-04 12:47:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12767)
[2020-07-04 12:47:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12768)
[2020-07-04 12:47:49 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12770)
[2020-07-04 15:17:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12764)
[2020-07-04 15:17:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12766)
[2020-07-04 15:17:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13046)
[2020-07-04 15:17:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13047)
[2020-07-04 15:17:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13049)
[2020-07-04 15:17:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13050)
[2020-07-04 15:17:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13051)
[2020-07-04 15:33:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:12769)
[2020-07-04 15:33:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13048)
[2020-07-04 15:33:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13262)
[2020-07-04 15:33:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13263)
[2020-07-04 15:33:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13265)
[2020-07-04 15:33:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13266)
[2020-07-04 15:33:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13267)
[2020-07-04 15:33:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13268)
[2020-07-04 16:42:28 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13264)
[2020-07-04 16:42:28 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13276)
[2020-07-04 16:42:28 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13277)
[2020-07-04 16:42:28 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13279)
[2020-07-04 16:42:28 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13280)
[2020-07-04 16:42:28 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13283)
[2020-07-04 18:10:37 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13278)
[2020-07-04 18:10:37 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13281)
[2020-07-04 18:10:37 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13282)
[2020-07-04 18:10:37 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13552)
[2020-07-04 18:10:37 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13553)
[2020-07-04 18:10:37 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13555)
[2020-07-04 18:10:37 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13556)
[2020-07-04 18:10:37 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13557)
[2020-07-04 18:41:25 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13558)
[2020-07-04 18:41:25 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13568)
[2020-07-04 18:41:25 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13569)
[2020-07-04 18:41:25 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13570)
[2020-07-04 18:41:25 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13577)
[2020-07-04 18:41:25 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13578)
[2020-07-04 20:29:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13571)
[2020-07-04 20:29:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13573)
[2020-07-04 20:29:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13737)
[2020-07-04 20:29:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13740)
[2020-07-04 20:29:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13741)
[2020-07-04 20:29:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13742)
[2020-07-04 20:29:29 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13743)
[2020-07-04 21:17:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13576)
[2020-07-04 21:17:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:13738)
[2020-07-04 21:17:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14030)
[2020-07-04 21:17:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14031)
[2020-07-04 21:17:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14032)
[2020-07-04 21:17:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14033)
[2020-07-04 21:17:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14034)
[2020-07-04 21:17:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14035)
[2020-07-04 21:17:50 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14036)
[2020-07-04 21:17:54 -0400] [14296] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 21:23:20 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14293)
[2020-07-04 21:23:20 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14294)
[2020-07-04 21:23:20 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14295)
[2020-07-04 21:23:20 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14296)
[2020-07-04 21:23:20 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14297)
[2020-07-04 21:23:20 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14298)
[2020-07-04 21:23:20 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14299)
[2020-07-04 21:23:20 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14300)
[2020-07-04 21:23:20 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14301)
[2020-07-04 21:23:23 -0400] [14309] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 21:23:23 -0400] [14309] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 21:26:27 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14309)
[2020-07-04 21:26:27 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14310)
[2020-07-04 21:26:27 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14311)
[2020-07-04 21:26:27 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14312)
[2020-07-04 21:26:27 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14313)
[2020-07-04 21:26:27 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14315)
[2020-07-04 21:26:27 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14316)
[2020-07-04 21:26:27 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14317)
[2020-07-04 21:53:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14314)
[2020-07-04 21:53:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14348)
[2020-07-04 21:53:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14349)
[2020-07-04 21:53:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14353)
[2020-07-04 21:53:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14354)
[2020-07-04 21:53:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14355)
[2020-07-04 21:53:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14356)
[2020-07-04 21:53:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14358)
[2020-07-04 21:53:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14359)
[2020-07-04 21:53:11 -0400] [14376] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 21:53:11 -0400] [14376] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-04 22:22:44 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14376)
[2020-07-04 22:22:44 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14377)
[2020-07-04 22:22:44 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14379)
[2020-07-04 22:22:44 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14380)
[2020-07-04 22:22:44 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14381)
[2020-07-04 22:22:44 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14383)
[2020-07-04 22:22:44 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14384)
[2020-07-04 22:22:44 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14386)
[2020-07-04 23:34:08 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14378)
[2020-07-04 23:34:08 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14391)
[2020-07-04 23:34:08 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14392)
[2020-07-04 23:34:08 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14393)
[2020-07-04 23:34:08 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14395)
[2020-07-04 23:34:08 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14396)
[2020-07-04 23:34:08 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14398)
[2020-07-05 00:10:36 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14394)
[2020-07-05 00:10:36 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14397)
[2020-07-05 00:10:36 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14751)
[2020-07-05 00:10:36 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14752)
[2020-07-05 00:10:36 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14753)
[2020-07-05 00:10:36 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14754)
[2020-07-05 00:37:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14749)
[2020-07-05 00:37:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14750)
[2020-07-05 00:37:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14755)
[2020-07-05 00:37:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14768)
[2020-07-05 00:37:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14770)
[2020-07-05 00:37:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14771)
[2020-07-05 00:37:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14772)
[2020-07-05 00:37:35 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14773)
[2020-07-05 01:54:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14774)
[2020-07-05 01:54:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14798)
[2020-07-05 01:54:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14800)
[2020-07-05 01:54:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14801)
[2020-07-05 01:54:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14802)
[2020-07-05 01:54:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14803)
[2020-07-05 01:54:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14804)
[2020-07-05 01:54:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14805)
[2020-07-05 02:43:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14817)
[2020-07-05 02:43:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14818)
[2020-07-05 02:43:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14822)
[2020-07-05 02:43:05 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14824)
[2020-07-05 02:43:07 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14821)
[2020-07-05 03:22:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14799)
[2020-07-05 03:22:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14820)
[2020-07-05 03:22:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14823)
[2020-07-05 03:22:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15114)
[2020-07-05 03:22:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15117)
[2020-07-05 03:22:21 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15119)
[2020-07-05 04:29:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:14819)
[2020-07-05 04:29:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15112)
[2020-07-05 04:29:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15113)
[2020-07-05 04:29:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15127)
[2020-07-05 04:29:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15128)
[2020-07-05 04:29:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15129)
[2020-07-05 04:29:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15130)
[2020-07-05 04:29:15 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15131)
[2020-07-05 05:32:33 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15132)
[2020-07-05 05:32:33 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15328)
[2020-07-05 05:32:33 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15329)
[2020-07-05 05:32:33 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15330)
[2020-07-05 05:32:33 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15332)
[2020-07-05 05:32:33 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15333)
[2020-07-05 07:17:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15494)
[2020-07-05 07:17:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15499)
[2020-07-05 07:17:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15327)
[2020-07-05 07:17:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15492)
[2020-07-05 07:17:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15498)
[2020-07-05 07:17:20 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15331)
[2020-07-05 07:54:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15326)
[2020-07-05 07:54:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15495)
[2020-07-05 07:54:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15877)
[2020-07-05 07:54:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15878)
[2020-07-05 07:54:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15882)
[2020-07-05 07:54:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15883)
[2020-07-05 07:54:24 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15886)
[2020-07-05 08:46:23 -0400] [15897] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-05 11:44:34 -0400] [15496] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-05 13:52:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15496)
[2020-07-05 13:52:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15893)
[2020-07-05 13:52:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15895)
[2020-07-05 13:52:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15896)
[2020-07-05 13:52:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15897)
[2020-07-05 13:52:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15898)
[2020-07-05 13:52:16 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15899)
[2020-07-05 13:58:01 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17194)
[2020-07-05 13:58:03 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15894)
[2020-07-05 13:58:03 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17198)
[2020-07-05 13:58:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17196)
[2020-07-05 13:58:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17199)
[2020-07-05 14:40:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:15884)
[2020-07-05 14:40:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17193)
[2020-07-05 14:40:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17197)
[2020-07-05 14:40:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17249)
[2020-07-05 14:40:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17252)
[2020-07-05 14:40:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17253)
[2020-07-05 14:40:04 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17256)
[2020-07-05 15:49:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17195)
[2020-07-05 15:49:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17255)
[2020-07-05 15:49:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17262)
[2020-07-05 15:49:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17263)
[2020-07-05 15:49:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17264)
[2020-07-05 15:49:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17265)
[2020-07-05 15:49:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17266)
[2020-07-05 15:49:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17267)
[2020-07-05 15:49:30 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17268)
[2020-07-05 15:49:37 -0400] [17602] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-05 18:22:42 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17597)
[2020-07-05 18:22:42 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17599)
[2020-07-05 18:22:42 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17600)
[2020-07-05 18:22:42 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17603)
[2020-07-05 18:22:42 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17604)
[2020-07-05 18:22:42 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17605)
[2020-07-05 18:22:42 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17607)
[2020-07-05 20:23:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17598)
[2020-07-05 20:23:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:17602)
[2020-07-05 20:23:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18133)
[2020-07-05 20:23:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18134)
[2020-07-05 20:23:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18136)
[2020-07-05 20:23:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18137)
[2020-07-05 20:23:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18138)
[2020-07-05 20:23:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18139)
[2020-07-05 20:23:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18141)
[2020-07-05 20:23:22 -0400] [18156] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-05 20:53:18 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18152)
[2020-07-05 20:53:18 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18154)
[2020-07-05 20:53:18 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18155)
[2020-07-05 20:53:18 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18156)
[2020-07-05 20:53:18 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18157)
[2020-07-05 20:53:18 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18158)
[2020-07-05 20:53:18 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18159)
[2020-07-05 20:53:18 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18160)
[2020-07-05 20:58:47 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18161)
[2020-07-05 20:58:49 -0400] [18171] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-05 20:58:52 -0400] [18173] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-05 21:02:23 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18167)
[2020-07-05 21:02:23 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18170)
[2020-07-05 21:02:27 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18168)
[2020-07-05 21:02:27 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18181)
[2020-07-05 21:34:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18169)
[2020-07-05 21:34:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18171)
[2020-07-05 21:34:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18172)
[2020-07-05 21:34:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18173)
[2020-07-05 21:34:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18174)
[2020-07-05 21:34:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18254)
[2020-07-05 21:34:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18259)
[2020-07-05 21:34:57 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18260)
[2020-07-05 22:44:56 -0400] [18274] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-06 01:39:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18253)
[2020-07-06 01:39:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18270)
[2020-07-06 01:39:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18271)
[2020-07-06 01:39:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18272)
[2020-07-06 01:39:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18273)
[2020-07-06 01:39:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18274)
[2020-07-06 01:39:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18275)
[2020-07-06 01:39:54 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18277)
[2020-07-06 03:41:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:18276)
[2020-07-06 03:41:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19627)
[2020-07-06 03:41:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19628)
[2020-07-06 03:41:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19629)
[2020-07-06 03:41:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19630)
[2020-07-06 03:41:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19633)
[2020-07-06 03:41:19 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19635)
[2020-07-06 04:00:10 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19631)
[2020-07-06 04:58:07 -0400] [19671] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-06 04:58:07 -0400] [19675] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-06 04:58:19 -0400] [19674] [ERROR] Socket error processing request.
Traceback (most recent call last):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 135, in handle
    self.handle_request(listener, req, client, addr)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 191, in handle_request
    six.reraise(*sys.exc_info())
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/six.py", line 625, in reraise
    raise value
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    resp.write_file(respiter)
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 403, in write_file
    if not self.sendfile(respiter):
  File "/Users/vignesh/anaconda3/lib/python3.7/site-packages/gunicorn/http/wsgi.py", line 393, in sendfile
    sent += sendfile(sockno, fileno, offset + sent, count)
OSError: [Errno 57] Socket is not connected
[2020-07-06 04:59:18 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19670)
[2020-07-06 04:59:18 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19671)
[2020-07-06 04:59:18 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19675)
[2020-07-06 05:37:55 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19634)
[2020-07-06 05:37:55 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19672)
[2020-07-06 05:37:55 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19674)
[2020-07-06 05:37:55 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19677)
[2020-07-06 05:37:55 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19716)
[2020-07-06 05:37:55 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19762)
[2020-07-06 05:37:55 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19763)
[2020-07-06 05:37:55 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19764)
[2020-07-06 08:15:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19676)
[2020-07-06 08:15:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19794)
[2020-07-06 08:15:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19795)
[2020-07-06 08:15:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19796)
[2020-07-06 08:15:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19798)
[2020-07-06 08:15:12 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19799)
[2020-07-06 08:55:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19792)
[2020-07-06 08:55:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19797)
[2020-07-06 08:55:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20129)
[2020-07-06 08:55:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20130)
[2020-07-06 08:55:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20131)
[2020-07-06 08:55:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20133)
[2020-07-06 08:55:31 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20134)
[2020-07-06 10:25:03 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:19793)
[2020-07-06 10:25:03 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20132)
[2020-07-06 10:25:03 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20166)
[2020-07-06 10:25:03 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20167)
[2020-07-06 10:25:03 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20168)
[2020-07-06 10:25:03 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20169)
[2020-07-06 10:25:03 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20170)
[2020-07-06 10:25:03 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20171)
[2020-07-06 10:25:03 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20172)
[2020-07-06 11:41:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20197)
[2020-07-06 11:41:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20198)
[2020-07-06 11:41:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20199)
[2020-07-06 11:41:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20200)
[2020-07-06 11:41:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20201)
[2020-07-06 11:41:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20202)
[2020-07-06 11:41:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20203)
[2020-07-06 11:41:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20204)
[2020-07-06 11:41:13 -0400] [9900] [CRITICAL] WORKER TIMEOUT (pid:20205)
